TopsortAnalytics/src/main/java/com/topsort/analytics/banners/BannerConfig.kt

```diff
@@ -0,0 +1,84 @@
+package com.topsort.analytics.banners
+
+import com.topsort.analytics.model.auctions.Device
+
+/**
+ * Class that handles different type of Banner configurations
+ */
+sealed class BannerConfig private constructor() {
+
+    /**
+     * Banner configuration for landing page banners
+     *
+     * @property slotId id of the banner slot
+     * @property ids ids of the entities that are competing for the banner
+     * @property device target device for the banner
+     * @property geoTargeting optional location for geo-targeted banners
+     */
+    data class LandingPage(
+        val slotId: String,
+        val ids: List<String>,
+        val device: Device = Device.MOBILE,
+        val geoTargeting: String? = null
```

Comment:
Does this convert to `geoTargeting: { location: string }` in the final payload?

Reply:
yes, the `build*` functions in the `Auction` class do that for you :)